src/main/java/edu/harvard/iq/dataverse/MailServiceBean.java

```diff
@@ -177,11 +177,11 @@ public InternetAddress getSystemAddress() {
     }
 
     //@Resource(name="mail/notifyMailSession")
-    public void sendMail(String from, String to, String subject, String messageText) {
-        sendMail(from, to, subject, messageText, new HashMap<>());
+/*    public void sendMail(String from, String to, String cc, String subject, String messageText) {
+        sendMail(from, to, cc, subject, messageText, new HashMap<>());
     }
-
-    public void sendMail(String reply, String to, String subject, String messageText, Map<Object, Object> extraHeaders) {
+*/
+    public void sendMail(String reply, String to, String cc, String subject, String messageText, Map<Object, Object> extraHeaders) {
         try {
```

Comment:
The only two usages I see for this method pass null for the extra headers param. Is it needed?

Reply:
OK. It looks like it's been around forever. hmm

Reply:
Removed commented-out code and remove the extra param and code here.